src/components/input-panel/spec-editor/renderer.js

```diff
@@ -65,8 +65,7 @@ export default class Editor extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-
-    if (nextProps.parse) {
+    if (!nextProps.autoParse && nextProps.parse) {
```

Comment:
@domoritz This is unrelated to this bug, but it makes our app perform better. Previously we updated the spec twice if it's in auto parse mode, since both `handleEditorChange` and `componentWillReceiveProps` are called. Now code inside `componentWillReceiveProps`will only be called in manual parse mode.